doozerlib/operator_metadata.py

```diff
@@ -267,6 +268,18 @@ def create_metadata_dockerfile(self):
             self.stream)
         del(metadata_dockerfile.labels['release'])
 
+    def create_container_yaml(self):
+        """Use container.yaml to disable unnecessary multiarch"""
+        filename = "{}/{}/container.yaml".format(self.working_dir, self.metadata_repo)
+        with open(filename, "w") as cyml:
```

Comment:
Use `io.open()` instead of builtin `open()` until we drop support for Python 2.7. And we can also mock `io` module to make test easier.